mkdocs.template.yml

```diff
@@ -80,6 +81,68 @@ plugins:
         "configuration/files.md": "concepts/configuration-files.md"
         "configuration/indexes.md": "concepts/indexes.md"
         "configuration/environment.md": "reference/environment.md"
+  - llmstxt:
+      markdown_description: |
+        You can use uv to install Python dependencies, run scripts, manage virtual environments,
+        build and publish packages, and even install Python itself. uv is capable of replacing
+        `pip`, `pip-tools`, `pipx`, `poetry`, `pyenv`, `twine`, `virtualenv`, and more.
+
+        uv includes both a pip-compatible CLI (prepend `uv` to a pip command, e.g., `uv pip install ruff`)
+        and a first-class project interface (e.g., `uv add ruff`) complete with lockfiles and
+        workspace support.
+      sections:
+        Getting started:
+        - getting-started/installation.md
```

Comment:
Should we be adding titles to the links? They are sometimes clearer than the filename.